ghost/i18n/locales/pt-BR/comments.json

```
@@ -6,20 +6,20 @@
     "{{amount}} more": "{{amount}} mais",
     "1 comment": "1 comentário",
     "Add comment": "Adicionar comentário",
-    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Adicione contexto ao seu comentário, compartilhe seu nome e expertise para promover uma discussão saudável.",
+    "Add context to your comment, share your name and expertise to foster a healthy discussion.": "Adicione contexto ao seu comentário, compartilhe seu nome e sua especialidade para promover uma discussão saudável.",
     "Add reply": "Adicionar resposta",
     "Already a member?": "Já é um membro?",
     "Anonymous": "Anônimo",
-    "Are you sure?": "",
+    "Are you sure?": "Você tem certeza?",
     "Become a member of {{publication}} to start commenting.": "Torne-se um membro de {{publication}} para começar a comentar.",
     "Become a paid member of {{publication}} to start commenting.": "Torne-se um membro pago de {{publication}} para começar a comentar.",
     "Cancel": "Cancelar",
     "Comment": "Comentário",
```

Hey, I just noticed this mistake in the translation.
In English, here it is a verb and not a noun. Instead of "Comentário", it should be "Comentar"

```suggestion
    "Comment": "Comentar",
```
https://github.com/TryGhost/Ghost/blob/2ef9bf4ea242b9800302b74fa06574e26c52e0c3/ghost/i18n/locales/context.json#L42C25-L42C71

---

Marked as not yet ready to merge.  Please at me when ready and I'll put it back in the queue.

---

Resolved @cathysarisky @zenditimo